lib/libutee/tee_api_operations.c

```diff
@@ -1242,7 +1242,7 @@ TEE_Result TEE_MACCompareFinal(TEE_OperationHandle operation,
 			       const void *mac, uint32_t macLen)
 {
 	TEE_Result res;
-	uint8_t computed_mac[TEE_MAX_HASH_SIZE];
+	uint8_t computed_mac[TEE_MAX_HASH_SIZE] = { 0 };
```

Comment:
This is change in generic libutee. Should be part of separate commit.

Reply:
Wondering if this is a good change or not. In theory there should not be a problem as only valid bytes are being accessed due to size checks.

Guideline is to initialize variables to good initial state which says that we should in example zeroize it.

But looking around of this there also other variables that do not get initialized.

@jenswi-linaro & @jforissier should we be concerned? And do batch changes for those in another PR?

As far as I can see this does not have affect for this particular PRs functionality.